lib/matplotlib/axes/_axes.py

```diff
@@ -2988,7 +3004,14 @@ def extract_err(err, data):
             if noxlims.any():
                 yo, _ = xywhere(y, right, noxlims & everymask)
                 lo, ro = xywhere(left, right, noxlims & everymask)
-                barcols.append(self.hlines(yo, lo, ro, **eb_lines_style))
+
+                if border is not None:
+                    barcols.append(self.hlines(yo, lo, ro,
```

Comment:
I think this breaks things because `set_path_effects` return `None` not the object it is acting on.